### ISSUE TITLE
[ceph] overall_status HEALTH_WARNING not handled

### ISSUE THREAD

**Issue by ccocsas** [DataDog/dd-agent#2717]
ceph status can report 3 states, HEALTH_OK, HEALTH_WARNING or HEALTH_CRITICAL.  The ceph.py check only looks for the first.

```
            if s_status.find('_OK') != -1:
                status = AgentCheck.OK
            else:
                status = AgentCheck.CRITICAL
```
This works in my branch:

```
            if s_status.find('_OK') != -1:
                status = AgentCheck.OK
            elif s_status.find('_WARN') != -1:
                status = AgentCheck.WARNING
            else:
                status = AgentCheck.CRITICAL
```

**Comment by masci** [comment]
Hi @ccocsas and thanks for reporting this!
Do you mind to open a PR from your working branch? Thanks in advance :bow:

**Comment by ccocsas** [comment]
Sure, I just need to get it up to spec.

**Comment by olivielpeau** [comment]
Fixed in #2929, thanks @ccocsas!
